osu.Game/Online/API/Requests/SearchBeatmapSetsRequest.cs

```diff
@@ -62,4 +73,38 @@ public enum BeatmapSearchCategory
         [Description("My Maps")]
         Mine,
     }
+
+    public enum BeatmapSearchGenre
+    {
+        Any,
+        Unspecified,
+
+        [Description("Video Game")]
+        Game,
```

Comment:
I don't think the names of these enums matter when mapping requests, so I'd just keep casing as consistent with the descriptions as possible.
```suggestion
        VideoGame,
```